scipy/linalg/decomp.py

```diff
@@ -1267,3 +1267,138 @@ def hessenberg(a, calc_q=False, overwrite_a=False, check_finite=True):
     q, info = orghr(a=hq, tau=tau, lo=lo, hi=hi, lwork=lwork, overwrite_a=1)
     _check_info(info, 'orghr (hessenberg)', positive=False)
     return h, q
+
+
+def cdf2rdf(w, v):
+    """
+    Converts complex eigensystems to real eigenvalues in a block diagonal form
+    and the associated real eigenvectors.
+
+    If the eigensystem:
+
+    `w, v = scipy.linalg.eig(X)`
```

Comment:
For code blocks you can double the colons at the end and then leave 4 whitespaces before the code block lines See `eig` docstring in the same file.